cognite_toolkit/cdf_tk/load.py

```diff
@@ -855,12 +863,46 @@ def create(self, items: Sequence[T_Resource], drop: bool, filepath: Path) -> T_R
                         if item.external_id == ext_id:
                             items.remove(item)
                 try:
-                    return ExtractionPipelineList(self.client.extraction_pipelines.create(items))
+                    extractionPipelineList = ExtractionPipelineList(self.client.extraction_pipelines.create(items))
                 except Exception as e:
                     print(f"[bold red]ERROR:[/] Failed to create extraction pipelines.\n{e}")
                     self.ToolGlobals.failed = True
                     return None
-            return None
+
+        file_name = filepath.stem.split(".", 2)[1]
+        config_file_stem = f"{file_name}.config"
+        config_file = next(
+            (file for file in Path(filepath.parent).iterdir() if file.is_file() and config_file_stem in file.name),
```

Comment:
Just a suggestion to make it more precise 
```suggestion
            (file for file in Path(filepath.parent).iterdir() if file.is_file() and file.stem.endswith(config_file_stem)),
```